docs/DeveloperGuide.md

```diff
@@ -286,20 +286,32 @@ Use case ends.
 
 3a. The given index is invalid
 
-> 3a1. AddressBook shows an error message <br>
+> 3a1. Task Manager shows an error message <br>
   Use case resumes at step 2
 
-{More to be added}
+#### Use case: Search task
+
+**MSS**
+
+1. User type the correct command in the command box(search KEYWORD)
+2. Task Manager shows a task or more tasks which match the KEYWORD.<br>
+Use case ends.
+
+**Extensions**
+
+1a. The user don't type the correct keyword
+
+> Use case ends
+ 
+ 
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java `1.8.0_60` or higher installed.
-2. Should be able to hold up to 1000 persons.
+2. Should be able to hold up to 1000 tasks.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
```

Comment:
this shouldn't be here anymore if you guys are working towards making your command input to be more natural